src/models/address.js

```diff
@@ -5,4 +5,34 @@ import EasyPostObject from './easypost_object';
  * @public
  * @extends EasyPostObject
  */
-export default class Address extends EasyPostObject {}
+export default class Address extends EasyPostObject {
+  static street1;
+
+  static street2;
```

Comment:
Thought: I hate that our styleguide requires this...

Reply:
Per our internal discussion. We will disable `lines-between-class-members` in ESlint for now. We will fix the style guide in another PR.